### ISSUE TITLE
Failed to instantiate component or class "gridManager".

### ISSUE THREAD

**Issue by githubjeka** [bupy7/yii2-grid#1]
I just write
```php
<?= \bupy7\grid\GridView::widget([       
        'dataProvider' => $dataProvider,
        'filterModel' => $searchModel,
}
?>
```

**Comment by bupy7** [comment]
Yes. It's a bug. Just you should add gridManager =) https://github.com/bupy7/yii2-grid#adding-ability-resize-width-of-columns

**Comment by bupy7** [comment]
Need upgrade README.md where will wrote that need will configure a `gridManager`.

**Comment by bupy7** [comment]
Fix in 1.1.6 version.
